README.md

```diff
@@ -156,6 +156,14 @@ class MyView(ListView):
         return context
 ```
 
+# URL caching
+
+In order to reduce database load, there is a caching mechanism for URL model. It's enabled be default and will use Django's cache backend. To disable it (e.g. in tests), set `FINGERPRINT_URL_CACHE_KEY` to `None` in your settings.
```

Comment:
please use `FINGERPRINT_CACHE` which value refers to a key of CACHES setting as @agoncharov-reef suggested

It is much more universal that way.

Your implementation doesn't allow me to use a different type of cache for FINGERPRINT app, instead I would be forced to change `"default"` django cache...